src/main/java/mate/controller/GetAllManufacturerController.java

```diff
@@ -0,0 +1,27 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Manufacturer;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/manufacturers/all")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/manufacturers")
```
Те ж саме і для інших get all запитів

Reply:
Почему, если гэт и all разные запросы?и ссылаются на разные jsp?